<!--Read external file and publish it on a topic using point cloud msgs -->


<launch>

<param name="bloomLoc_fpath"    type="string"   value="/home/thivanka/catkin_ws/src/uri_soft_wip/cyanobloom_sabbatus/Data_files/drew_results/2016_modified/" />
<param name="init_filename"     type="string"   value="bloom_coordinates_wind_drift0.txt" />
<param name="bloom_topic"       type="string"   value="particle_list_for_gazebo" />

<!--should equal to number of list of blooms/-->
<param name="cycles"            type="int"      value="5"/>
<!--node to read external files/-->
<node   name    = "blooms_publisher_node" 
        pkg     = "cyanob_phd_filter_sabbatus"  
        type    = "cyanob_phd_filter_sabbatus_blooms_publisher_node" 
        output  = "screen"   
/>



</launch>


